docs/tpm.md

```diff
@@ -273,7 +273,7 @@ Find a persistent storage index that is unused in the TPM and note it down. It u
 # 5. Issue the command 
 
   $ make 
-  (You may need to run �sudo apt install make gcc� before running this command successfully).
+  (You may need to run sudo apt install make gcc before running this command successfully).
```

Comment:
Rendering issue of some special character before and after `sudo apt install make gcc` ??

Reply:
Done